lib/Utils.ts

```diff
@@ -154,3 +154,12 @@ export const groupBy = (arr: object[], keyGetter: (item: any) => string | number
 export const ms = (): number => {
   return Date.now();
 };
+
+/**
+ * Split listen string from config files.
```

Comment:
Maybe mention that the string is split in the host and port. And the source of that string doesn't necessarily have to be a config file. I could think of other situations in which we could use this method.